src/components/FilterBar/SavedSearchesList/SavedSearchesList.react.js

```diff
@@ -0,0 +1,57 @@
+import {SavedSearchesListItem} from './SavedSearchesListItem.react';
+
+export class SavedSearchesList extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = this.getStateFromStores();
+    this.props.filterBarStore.addChangeListener(this._onChange.bind(this));
```

Comment:
Yeah, I agree. I'll move all the search stuff into it's own store and actor now.